.pre-commit-config.yaml

```diff
@@ -11,3 +11,8 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
+        args: ["--profile", "black"]
```

Comment:
forgot this about curve-ball: isort and black can be in conflict without this

Reply:
Good catch!